djangoplicity/pages/tests/test_models.py

```diff
@@ -0,0 +1,150 @@
+from django.conf import settings
+from datetime import datetime, timedelta
+from django.test import TestCase
+from factories import SectionFactory, PageFactory, URLFactory, PageGroupFactory, EmbeddedPageKeyFactory
+from djangoplicity.pages.models import (
+    Page, URL, EmbeddedPageKey, Section,
+    register_page_key, generate_page_id, page_cache_invalidator, 
+    url_cache_invalidator, page_key_cache_invalidator, section_cache_invalidator
+    )
+
+from mock import patch
```

Comment:
Are you using this import?.

Reply:
Resolved in [#51b12dc](https://github.com/djangoplicity/djangoplicity/pull/96/commits/51b12dc847fc89988959c6c94d0a40b700743c6b)